code/Employee.php

```diff
@@ -142,7 +145,7 @@ public function validate()
     {
         $result = parent::validate();
         if (!$this->Name) {
-            $result->error('"Name" can\'t be blank');
+            $result->addFieldError('Name', '"Name" can\'t be blank');
```

Comment:
This was just plum broken. There's no such method as `error()` on `ValidationResult`.